### PR TITLE
add troubleshooting section to contour

### PR DESCRIPTION
**Pull request by KauzClay**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)? 1-8 (though idk if that branch is ready yet)

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: contour/how-to-guides/troubleshooting.hbs.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### Explanation
+
+By default, the Contour package progams Envoy's listeners to use IPv6 addresses. In most cases, this is okay. However, some clusters (such as TKGI) only support IPv4. 
+
+### Solution
+
+>**Important** This solution only applies to Contour 1.26 or newer, which is included in Tanzu Application Platform v1.8.0 and later.
+
+To switch to using IPv4 addresses on the listeners, you just need to update the Tanzu Application Platform values file (commonly called `tap-values.yaml`).
```
Comment (andrew-su) [inline contour/how-to-guides/troubleshooting.hbs.md:23]:
```suggestion
To switch to using IPv4 addresses on the listeners, update the Tanzu Application Platform values file (commonly called `tap-values.yaml`).
```

File: contour/how-to-guides/troubleshooting.hbs.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+The following is an example snippet of the Contour section of the values file:
```
Comment (andrew-su) [inline contour/how-to-guides/troubleshooting.hbs.md:25]:
```suggestion
The following is an example of the Contour section of the values file:
```

File: contour/how-to-guides/troubleshooting.hbs.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+The Contour package fails to reconcile, and the Envoy pods fail with errors like:
+
+```
+23-11-23 01:55:11.503][1][warning][config] [source/common/config/grpc_subscription_impl.cc:128] gRPC config for type.googleapis.com/envoy.config.listener.v3.Listener rejected: Error adding/updating listener(s) ingress_http: malformed IP address: :: 
+```
+
+### Explanation
+
+By default, the Contour package progams Envoy's listeners to use IPv6 addresses. In most cases, this is okay. However, some clusters (such as TKGI) only support IPv4.
```
Comment (jsanin-vmw) [inline contour/how-to-guides/troubleshooting.hbs.md:17]: typo: programs

**Review by andrew-su — APPROVED** [review]
LGTM!

**Pull request merged**